src/status.cc

```diff
@@ -125,6 +125,14 @@ std::string getStatusString(Status status) {
       return "X509 parse pubkey fails";
     case Status::JwksX509GetPubkeyError:
       return "X509 parse pubkey internal fails: get pubkey";
+
+    case Status::PKCSNotImplementedKty:
+      return "Key type is not supported";
+    case Status::PKCSPemParseError:
+      return "PKCS pubkey parse fails";
+
+    case Status::BioAllocError:
+      return "Failed to create BIO";
```

Comment:
Failed to create BIO due to memory allocation failure"

Reply:
done